ariadne/cardano/src/Ariadne/Wallet/Backend/KeyStorage.hs

```diff
@@ -336,19 +336,13 @@ addWallet ::
 addWallet pwl WalletFace {..} esk mbWalletName utxoByAccount hasPP createWithA assurance = do
   walletName <-
       fromMaybe
-      (genWalletName <$> pwlGetDBSnapshot pwl)
+      (pure $ WalletName "Untitled wallet")
       (pure <$> mbWalletName)
```

Comment:
Now both cases are `pure`, so it can be simplified to `let walletName = fromMaybe "Untitled wallet" mbWalletName`.

Reply:
Note that `WalletName` is instance of `IsString`, so applying `WalletName` constructor is not necessary (but it's not a big deal and I don't mind doing so).

Reply:
Good catch. Will fix ASAP.